README.md

```diff
@@ -1,12 +1,16 @@
 # Explainer: Tightening HTTP State Management
 
-Mike West, August 2018
+forked from
 
-_©2018, Google, Inc. All rights reserved._
+[Mike West](https://github.com/mikewest/http-state-tokens), August 2018 to provide a pull request.
```

Comment:
(Disclaimer: I'm not a project member.)
I guess the above forked from texts are not supposed to get merged with this PR?